velox/exec/tests/AggregationRunnerTest.cpp

```diff
@@ -76,8 +89,8 @@ int main(int argc, char** argv) {
   facebook::velox::aggregate::prestosql::registerAllAggregateFunctions();
   facebook::velox::functions::prestosql::registerAllScalarFunctions();
 
-  auto duckQueryRunner =
-      std::make_unique<facebook::velox::exec::test::DuckQueryRunner>();
+  auto queryRunner = facebook::velox::exec::test::setupReferenceQueryRunner(
```

Comment:
Could we still keep duckQueryRunner when presto_url is not set? We may need this since aggregation fuzzer test running internally is not using PQR yet.